### PR TITLE
Subscriptions model, dao, and controller

### PR DESCRIPTION
**Pull request by ChenJesse**
screen shot 2017-09-03 at 6 06 56 pm
screen shot 2017-09-03 at 6 08 05 pm
screen shot 2017-09-03 at 6 08 24 pm
screen shot 2017-09-03 at 6 08 53 pm

Where my `user_id=4`, there were two fake series I created with `series_id=2` and `series_id=3`

Decided to just put in the required `series` data as fields in the `subscriptions` model, since I would have had to make other queries if we ever retrieve a subscription. Code would have been messy (b/c they're in different databases) and more expensive (I think it's the denormalization concept you were talking about?)

### CODE REVIEW

**Review by kevingreer — APPROVED** [review]
Seems good to me. My only concern is that if a series were to ever update a field then we'd need to update every subscription that involves that series. That might be rare enough to be worth it though.

Also my PR is ready to go but I'll wait until you merge this to put it up.

**Comment by Jma353** [comment]
I agree with @kevingreer.  It might make more sense to remove `series`-specific fields from the subscription and then, in the `dao` grab the subscriptions, grab the series, have a non-db field in `Subscription` called "series", and then set that field on grabbing both sets of information.  Does that make sense?  We should keep denormalization (duplication of data) to a minimum with regards to podcasts data, b/c it's so large.

File: py/src/app/pcasts/controllers/series_subscriptions_controller.py
Comment (kevingreer) [inline py/src/app/pcasts/controllers/series_subscriptions_controller.py]: the apiary documentation has this key as singular "subscription" since there's only one, same for the delete method

Comment (ChenJesse) [reply py/src/app/pcasts/controllers/series_subscriptions_controller.py]: ahh good catch

**Comment by ChenJesse** [comment]
screen shot 2017-09-04 at 2 26 01 pm
screen shot 2017-09-04 at 2 24 56 pm
screen shot 2017-09-04 at 2 24 35 pm
screen shot 2017-09-04 at 2 24 08 pm

Where my user id = 8, series_id = 1

**Comment by Jma353** [comment]
🚢

**Pull request merged**
